ftl_installer/config/Config.py

```diff
@@ -0,0 +1,37 @@
+import yaml
+
+
+class Config(object):
+    """
+    Class to handle the configuration given with `--config` option.
+    """
+
+    def __init__(self):
+        """
+        generate an empty config when no input config file provided.
+        """
+
+        self.config = ""
+
+    def config_from_file(self, config_file):
+        """
+        stores the configuration from file to a config object.
+        """
+
+        with open(config_file) as config_file:
+            self.config = yaml.safe_load(config_file)
+
+    def write_config(self):
+        """
+        writes the config object to a file in block format.
+        """
+
+        with open('config.yml', 'w') as outfile:
```

Comment:
:+1: to the `__init__` approach. We'll want to use `~/.config/ftl_installer/ftl_installer.yml` by default, but that should be passed into the config class so that we can make the default file location configurable at some point in the future.